plugins/outputs/bigquery/bigquery.go

```diff
@@ -0,0 +1,215 @@
+package bigquery
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"reflect"
+
+	"cloud.google.com/go/bigquery"
+	"github.com/influxdata/telegraf"
```

Comment:
You might want to put this one in a separate "group" below as it is an "internal" package.